bridgetown.automation.rb

```diff
@@ -1,15 +1,26 @@
 say_status :plausible, "Installing the bridgetown-plausible plugin..."
 
 domain_name = ask("What's your Plausible domain?")
-
+server_name = ask("What's your Plausible server? Leave blank if plausible.io")
```

Comment:
the reasoning goes as such:
it's an automation, so you want it to take care of all the reasonable config options, especially in a setting where there are only two really.

i personally do not use automations because i just prefer the handywork to understand what's going on. but if i would ever use it, i want it to take care of things.

hence your wordy explanation makes a lot of sense, because for a selfhoster, it satisfies the need for configuration, and for a 'standard' user, it properly explains the option without confusion. maybe: " leave blank if you are using plausible.io services" ?

however, i'm really not married to this, your alternative option 1 also seems reasonable.

this might just be one of these let's-choose-one-and-stick-with-it-until-somebody-else-complains-scenarios?